test/100-basic-name-resolution.bats

```diff
@@ -17,6 +17,18 @@ load helpers
 	assert "$ip_a1"
 }
 
+@test "basic container - dns itself with long network name" {
+	subnet_a=$(random_subnet 5)
+	create_config "bignetttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttworkname" $(random_string 64) "aone" "$subnet_a" "a1" "1a"
```

Comment:
Please make the name a variable and use this instead of having the full name three times, this is unreadable